source/openapi-admin-v3.yaml

```diff
@@ -1701,12 +1701,23 @@ paths:
     parameters:
       - "$ref": "#/components/parameters/GroupId"
       - "$ref": "#/components/parameters/AppId"
+      - in: query
+        name: after
+        required: false
+        schema: 
+          type: string
+        description: |-
+          The unique ``_id`` for a pending user. List pending users can return 50
```

Comment:
A suggestion (only) that I think makes it a bit more scan-able:

To view additional results, use the ``_id`` of the last pending user listed 
 in the response to call ``list pending users`` again, passing the _id it to the ``after`` parameter.